cmd/world/root/root.go

```diff
@@ -44,6 +52,63 @@ func init() {
 	logger.AddLogFlag(doctorCmd)
 }
 
+func checkLatestVersion() error {
+	url := "https://api.github.com/repos/Argus-Labs/world-cli/releases/latest"
+
+	// Create a request with the context
+	req, err := http.NewRequestWithContext(context.Background(), http.MethodGet, url, nil)
+	if err != nil {
+		return fmt.Errorf("error creating request: %s", err.Error())
+	}
+
+	// Create a new HTTP client and execute the request
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	if err != nil {
+		return fmt.Errorf("error fetching the latest release: %s", err.Error())
+	}
+	defer resp.Body.Close()
+
+	// Release structure to hold the data of the latest release
+	type Release struct {
+		TagName string `json:"tag_name"`
+		Name    string `json:"name"`
+		HTMLURL string `json:"html_url"`
+	}
+
+	// Decode the JSON response into the Release struct
+	var release Release
+	if err := json.NewDecoder(resp.Body).Decode(&release); err != nil {
+		return fmt.Errorf("error decoding the release data: %s", err.Error())
+	}
+
+	if AppVersion != "" {
+		v1, err := version.NewVersion(AppVersion)
+		if err != nil {
+			return fmt.Errorf("error parsing version 1: %s", err.Error())
+		}
+
+		v2, err := version.NewVersion(release.TagName)
+		if err != nil {
+			return fmt.Errorf("error parsing version 2: %s", err.Error())
+		}
+
+		if v1.LessThan(v2) {
+			colorReset := "\033[0m"
```

Comment:
use the lipgloss library for colors

https://github.com/charmbracelet/lipgloss

Reply:
updated